modules/logging/logging.go

```diff
@@ -35,8 +37,19 @@ func SharedInstance() *LogStruct {
 	return sharedInstance
 }
 
-func (u *LogStruct) MethodInfo(model string, method string) *logrus.Entry {
+func (u *LogStruct) MethodInfo(model string, method string, stack ...bool) *logrus.Entry {
```

Comment:
__[golint]__
* [golint] exported method LogStruct.MethodInfo should have comment or be unexported